src/scss/grommet-core/_objects.tile.scss

```diff
@@ -44,7 +44,18 @@
     }
   }
 
+  .#{$grommet-namespace}tiles__masonry-column {
+    > .#{$grommet-namespace}tile {
+      @include media-query(lap-and-up) {
+        min-width: $tile-size;
+        max-width: $tile-size;
```

Comment:
Setting flex-basis to $tile-size causes the heights of the Tiles to be equal for each row (when we want varying tile heights to be preserved for masonry layouts):
![screen shot 2016-08-01 at 3 46 58 pm](https://cloud.githubusercontent.com/assets/10161095/17314853/117dccfc-5803-11e6-87d3-e10cdce12ee6.png)